var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -337,6 +342,7 @@ class Trilinos(CMakePackage):
     depends_on('metis@5:', when='+metis')
     depends_on('suite-sparse', when='+suite-sparse')
     depends_on('zlib', when="+zlib")
+    depends_on('kokkos-nvcc-wrapper', when='+cuda')
```

Comment:
Does this work?  I've been playing around with this locally and I'm not getting `nvcc-wrapper` set as my compiler. Seems like I still need to manually add `define(`CMAKE_CXX_COMPILER`, path_to_nvcc_wrapper)`

Reply:
@psakievich I think you are right. I don't see anyone using `kokkos-nvcc-wrapper` as dependency, as most use it as part of CMake options. I'm not sure why it was added then.

Reply:
Are you using MPI? kokkos-nvcc-wrapper should set environment variables that control which underlying compiler is picked up by MPI.

Reply:
@amklinv-nnl I'm using MPI.  I was using inheritance to reconstruct this PR in a repo though so I could have missed something.  I'll take another look.